css/three-bounce.css

```diff
@@ -1,3 +1,7 @@
+.sk-three-bounce {
+  height: 18px;
+}
```

Comment:
user-agent default styles added padding to the bottom of this div that affected proper vertical alignment.